src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java

```diff
@@ -1,23 +1,32 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- *
+ * <p>
  * This software is the confidential and proprietary information of Fundacion Jala
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
  */
 package org.fundacion.jala.converter.controller;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.fundacion.jala.converter.service.ExtractMetadata;
 import org.fundacion.jala.converter.service.FileStorageService;
+import org.fundacion.jala.converter.service.ObjectMetadata;
+import org.fundacion.jala.converter.service.metadata.TypeFileExport;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+

```

Comment:
these spaces in the code should be removed

Reply:
ok